src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -98,6 +114,15 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
             list($alias, $field) = $this->addJoinsForNestedProperty($property, $alias, $queryBuilder, $queryNameGenerator);
         }
 
+        if (null !== $nullsComparison = $this->properties[$property]['nulls_comparison'] ?? null) {
+            $nullsDirection = (self::NULLS_SMALLEST === $nullsComparison) ? $direction : ('ASC' === $direction ? 'DESC' : 'ASC');
```

Comment:
You're right. It's not needed.

http://php.net/manual/en/language.operators.precedence.php